### ISSUE TITLE
Outdated Code

### ISSUE THREAD

**Issue by cmurfy** [tomfin46/IFDB.bundle#13]
If tomfin46 is still around, I'd like them to take a crack at updating the code for this info scraper, as right now it's failing to find certain edits (i.e. TM2YC's Man of Steel: A Symbol of Hope). Last update was two years ago, it seems, so it hasn't been entirely forgotten.

**Comment by circulon** [comment]
@cmurphy02 
I recently just started adding some cool Fanedits to plex and found this plugin to not even be recognised in the latest PMS Version (1.41.3.9314) which was disappointing.

So I decides to have a go at fixing it.
It now works with 
- SSLV3 security requirements
- The updated layouts of fanedit.org's search results and entry listings

Note: This is a WIP and I have only done basic testing with a few movies but it looks good so far

You can find  it here : https://github.com/circulon/IFDB.bundle.git

Enjoy

**Comment by cmurfy** [comment]
@circulon 
Wasn't expecting a response from anyone, given how long ago this agent was created.

So good to know there's a new one available! Only problem is, there appear to be no installation instructions available.

**Comment by circulon** [comment]
@cmurfy 
Yeah Installation is the same for all plex Plugins though as they all live in the same place.

Here is the docs from FanEdit https://ifdb.fanedit.org/how-to-add-fanedit-metadata-to-your-plex-media-server/

But in general terms you  need to find where Plex stores its data  in your os.
The plugins path on your host os will contain this segment `Library/Application\ Support/Plex\ Media\ Server/Plug-ins` 
This is a linux pathspec but the directore naming is the same regardless of OS

Once you have found the Plugins folder extract the zip or clone the repo into the `Plig-ins` folder which should create m `IFDB.bundle` folder.

restart PMS and you should be good to go.

I am owrking on the installation instructions to add to the README

Cheers

**Comment by circulon** [comment]
@cmurfy 
Have just added installation instructions to the README

**Comment by greenturnedblue** [comment]
This is great thanks Circulon! I just matched about 20 edits I've added to Plex since the old plugin stopped working and your new plugin worked perfectly. 

For reference, this is IFDB's guide how to install the old plugin. but the steps are identical https://fanedit.org/how-to-add-fanedit-metadata-to-your-plex-media-server/

**Comment by cmurfy** [comment]
@circulon
Excellent! I've already copied the IFDB.bundle folder to the plug-ins folder. Just a tad confused about the "Clone the repo" bit in the instructions.

**Comment by circulon** [comment]
> [@circulon](https://github.com/circulon) Excellent! I've already copied the IFDB.bundle folder to the plug-ins folder. Just a tad confused about the "Clone the repo" bit in the instructions.

NOTE that the instructions say "Install the IFDB Plug-in via one of the following methods"
using `git` to clone this repo is just one of the options.
As you got the bundle in place bia the .zip using the git clone is not necessary

**Comment by CAL0901** [comment]
Hi. Is there a way to use this on a Jellyfin server?

**Comment by circulon** [comment]
> Hi. Is there a way to use this on a Jellyfin server?

I Dont know but I would not think so.
Sorry

**Comment by CAL0901** [comment]
> > Hi. Is there a way to use this on a Jellyfin server?
> 
> I Dont know but I would not think so. Sorry

Okay. Thank you.
